tests/integration/states/test_vmc_distributed_firewall_rules.py

```diff
@@ -0,0 +1,192 @@
+"""
+    Integration Tests for vmc_distributed_firewall_rules state module
+"""
+import pytest
+import requests
+from saltext.vmware.utils import vmc_request
+
+
+@pytest.fixture
+def common_data(vmc_config):
+    data = vmc_config["vmc_nsx_connect"].copy()
+    data["domain_id"] = "default"
+    data["security_policy_id"] = "default-layer3-section"
```

Comment:
Not a blocker, but we should have the ability to list security policies. Took me a while to figure out that doesn't exist yet :sweat_smile: